@shared/api/internal.ts

```diff
@@ -367,17 +367,23 @@ export const signFreighterTransaction = async ({
   transactionXDR: string;
   network: string;
 }): Promise<{ signedTransaction: string }> => {
-  try {
-    const { signedTransaction } = await sendMessageToBackground({
-      transactionXDR,
-      network,
-      type: SERVICE_TYPES.SIGN_FREIGHTER_TRANSACTION,
-    });
-    return { signedTransaction };
-  } catch (e) {
-    console.error(e);
```

Comment:
@piyalbasu note: I switched these to not catch the errors here. They were getting caught too early and wouldn't trigger the thunk reject

Reply:
Cool - no problem. The try/catch's in the functions in this file are a little redundant anyway. Errors will be caught in the duck